src/components/Images.js

```diff
@@ -0,0 +1,187 @@
+        }, 3000)
+    }
+
+    const cleanSelectedImg = (previous, current) => {
+        setTimeout(() => {
+            setfinalGame(finalGame.map((img, index) => {
+                if(index === previous || index === current) {
+                    img.flipped = !img.flipped
+                }
+                  return {...img}
+              }))
+        }, 400)
+    }
+
+    const match = (previous, current) => {
+        setfinalGame(finalGame.map((img, index) => {
+            if(index === previous || index === current) {
+                img.completed = !img.completed
+            }
+              return {...img}
+          }))
+    }
+
+  const changeFlip = (ind, name, completed) => {
+
+    const flipImg = finalGame.map((img, index) => {
+        if(ind === index && !img.completed) {
+            img.flipped = !img.flipped
+        }
+          return {...img}
+      })
+      setfinalGame(flipImg)
+
+        if(!preName && !completed) {
+            setPreName(name)
+            setSelectedImg(ind)
+        } else if(preName === name && selectedImg != ind && !completed) {
+            setPoints(points + 1)
+            setPreName('')
+            match(selectedImg, ind)
+        } else if(preName !== name && !completed) {
+            setPreName('')
+            cleanSelectedImg(selectedImg, ind)
+        }
+
+    
+  }
+
+  const saveUserName = (e) => {
+      e.preventDefault()
+      setPlayerName(e.target.elements.user.value.charAt(0).toUpperCase() + e.target.elements.user.value.slice(1))
+      startGame()
+  }
+
+  const disableGame = {
+    pointerEvents: 'none',
+    opacity: '0.9'
+}
+
+    return (
+        <div className='container'>
+            <div className='text-center py-4'>
+                {hideStart && <h3>Your game starts in {timer}</h3>}
+                {playerName && <h3>{playerName} Your Points Is {points}</h3>}
+                {!playerName && <form onSubmit={saveUserName}>
+                <div className="row form-group">
+                    <label for="exampleInputEmail1">Your Name</label>
+                    <div className="row">
+                        <div className="col-3 mx-auto">
+                            <input type="text" name='user' className="form-control" placeholder="Enter Your Name" required/>
+                        </div>
+                    </div>
+                    <div className='col-3 mx-auto pt-2'><input type="submit" className='btn btn-primary' value="Start Game"/></div>
+                </div>
+                </form>}
+            </div>
+            {playerName && <div className="row">
+                {finalGame.map((img, index) => (
+                    <div className="col-2 "  >
+                        {/* style={disableGameDiv ? disableGame : ''} */}
+                        <img className='border img-thumbnail {img.completed ? disableGame : ``}' onClick={() => changeFlip(index, img.name, img.completed)} key={index} src={img.flipped ? img.loc : defaultImg} alt=""/>
```

Comment:
In disableGame you have mentioned few style props but you are using it as className??

I don't think it's working kindly verify